### PR TITLE
CSV changes to remove ocs metrics exporter deployment

### PR DESCRIPTION
**Pull request by umangachapagain**
We will deploy metrics exporter as part of StorageCluster reconcile now. We don't need it in CSV anymore.

### CODE REVIEW

**Comment by umangachapagain** [comment]
/hold
This depends on #2293 which brings rest of the implementation

File: controllers/storagecluster/exporter.go
Comment (jmolmo) [inline controllers/storagecluster/exporter.go:34]: now the comment is deprecated

**Review by jmolmo — COMMENTED** [review]
LGTM. I think we need to test this.
@umangachapagain Do we have test to get the list of metrics using the console ?

**Comment by umangachapagain** [comment]
/unblock
Testing of #2293 might be blocked due to resource conflicts with CSV. This PR is now updated to unblock it.

**Review by iamniting — APPROVED** [review]

**Review by iamniting — APPROVED** [review]

**Pull request merged**
